server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -43,6 +43,9 @@ export class SyncChefsDataService {
       submission: responseData.submission.data,
       confirmationId: responseData.confirmationId,
       facilityName: responseData.submission.data.facilityName,
+      projectTitle: responseData.submission.data.projectTitle,
+      totalEstimatedCost: responseData.submission.data.totalEstimatedCostOfProject,
+      asks: responseData.submission.data.totalRequestBeingMadeOfBcaapACDNotToExceedB,
       assignedTo: null,
```

Comment:
This should be removed. Since if it's already assigned to someone and we sync it, this value will be set to null

Reply:
Ah, makes total sense, removing these two.